src/components/ModelSteps/Scan.jsx

```diff
@@ -49,36 +49,38 @@ const Scan = ({ currentStep }) => {
     setIsImportFileModalOpen(true)
   }
 
-  const onFileChange = file => {
+  const onChangeFile = useCallback(file => {
     if (file) {
       setFile(file)
       setIsImportFileModalOpen(false)
     }
-  }
+  }, [])
 
   const openFilePickerModal = () => {
     setIsFilePickerModalOpen(true)
   }
   const closeFilePickerModal = () => setIsFilePickerModalOpen(false)
 
   const onChangeFilePicker = fileId => {
-    setFileId(fileId)
+    setCozyFileId(fileId)
   }
 
   useEffect(() => {
     ;(async () => {
-      if (fileId) {
-        const blobFile = await fetchBlobFileById(client, fileId)
+      if (cozyFileId) {
+        const blobFile = await fetchBlobFileById(client, cozyFileId)
         if (validFileType(blobFile)) {
-          setFile(blobFile)
+          // Set "id" to be able to delete the original file if needed
```

Comment:
I think we should test all those features instead of adding a single comment to document the code + prevent regression 🙏

Reply:
i think the test is not pushed :(